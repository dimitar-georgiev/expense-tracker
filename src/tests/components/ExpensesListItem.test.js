/**
 * Created by Mitaka on 26-Nov-17.
 */
import React from 'react';
import {shallow} from 'enzyme';
import ExpensesListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('render ExpenseListItem', () => {
    const wrapper = shallow(<ExpensesListItem {...expenses[0]}/>);

    expect(wrapper).toMatchSnapshot();
});