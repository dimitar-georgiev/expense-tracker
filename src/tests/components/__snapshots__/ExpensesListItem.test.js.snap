// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ExpenseListItem 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Test Description Edno
    </h3>
  </Link>
  <p>
    0
     - 
    0
  </p>
</div>
`;
